src/python/pants/backend/project_info/rules/fast_dependencies_test.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Change filename to `dependencies_test.py`. 

Also, this is still not fully decided but we started calling `self.request_single_product`-based tests (e.g. `ConsoleRuleTestBase`-based test) as `foo_integration_test.py`. It's not the same as a heavy-weight V1 integration test, but these are closer to integration test on the spectrum from unit tests to ITs. For example, rules that use Pex will end up making network requests. In contrast, `engine.util.run_rule`-based tests mock everything out and have fewer side effects.